README.md

```diff
@@ -16,15 +16,26 @@ Authors  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano (caggiano@gma
 
 `MyoSuite` is a collection of musculoskeletal environments and tasks simulated with the [MuJoCo](http://www.mujoco.org/) physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems.
 
- [Full task details](https://github.com/myohub/myosuite/blob/main/docs/source/suite.rst#tasks) | [Baselines](https://github.com/myohub/myosuite/tree/main/myosuite/agents/baslines_NPG) | [Documentation](https://myosuite.readthedocs.io/en/latest/)
-| [Tutorials](https://github.com/myohub/myosuite/tree/main/docs/source/tutorials)
+
+ <!-- TODO: add brieft history of the mujoco environment? -->
```

Comment:
@Vittorio-Caggiano @raku-slyu  - check the docs.

Reply:
removed on new PR